ledstripcontroller.py

```diff
@@ -0,0 +1,92 @@
+import uasyncio as asyncio
+from machine import Pin, PWM
+from encoder import Encoder
+
+
+class LedStripController:
+
+    def __init__(self, enc, button_pin = 14, fader_pins = (15,)):
```

Comment:
Later we should change this; we should just use Pin and PWM objects. Binding `button_pin` and `fader_pins` to the pin numbers will make it harder to port to the different MicroPython flavours - and a little harder to unit test.